packages/medusa-file-minio/src/services/minio.js

```diff
@@ -1,29 +1,28 @@
-import { AbstractFileService } from '@medusajs/medusa'
```

Comment:
question: isn't it something that is part of the minio PR here?

Reply:
Yes this is copied from the minio PR since I needed the functionality and I added a small change. Once the minio PR is merged this will resolve.